include/zephyr/net/wifi.h

```diff
@@ -104,17 +104,22 @@ static inline const char *wifi_mfp_txt(enum wifi_mfp_options mfp)
 	}
 }
 
-/** IEEE 802.11 operational frequency bands (not exhaustive). */
+/**
+ * @brief IEEE 802.11 operational frequency bands (not exhaustive).
+ */
 enum wifi_frequency_bands {
-	/** 2.4GHz band. */
+	/** 2.4 GHz band. */
```

Comment:
it would be good to have these doxygen changes in a separate commit.

Reply:
Ideally yes, but I felt that since these were related to the band extension changes I could just do it in that context.